api/main_endpoints/routes/Auth.js

```diff
@@ -84,6 +84,16 @@ router.post('/sendPasswordReset', async (req, res) => {
     if (!result) {
       return res.sendStatus(OK);
     }
+    if (result.accessLevel === membershipState.PENDING) {
+      return res.status(UNAUTHORIZED).send({
+        message: 'Email has not been verified.'
+      });
+    }
+    if (result.accessLevel === membershipState.BANNED) {
+      return res.status(UNAUTHORIZED).send({
+        message: 'User is banned.'
+      });
+    }
```

Comment:
we can use this to make it dynamic

https://github.com/SCE-Development/Clark/blob/0c66e2b03e2cef23f252e05b577f09e73090dbeb/src/Enums.js#L54


```suggestion
    if (
      [
        membershipState.PENDING,
        membershipState.BANNED,
      ].includes(result.accessLevel)
    ) {
      return res.status(UNAUTHORIZED).send(
        'Cannot reset password, account is in ' +
        membershipStateToString(result.accessLevel) +
        ' state.'
      );
    }
```

Reply:
done